models/review_test.go

```diff
@@ -105,3 +105,85 @@ func TestUpdateReview(t *testing.T) {
 	assert.NoError(t, UpdateReview(review))
 	AssertExistsAndLoadBean(t, &Review{ID: 1, Content: "Updated Review"})
 }
+
+func TestGetReviewersByPullID(t *testing.T) {
+	assert.NoError(t, PrepareTestDatabase())
+
+	issue := AssertExistsAndLoadBean(t, &Issue{ID: 2}).(*Issue)
+	user1 := AssertExistsAndLoadBean(t, &User{ID: 1}).(*User)
+	user2 := AssertExistsAndLoadBean(t, &User{ID: 2}).(*User)
+	user3 := AssertExistsAndLoadBean(t, &User{ID: 3}).(*User)
+	user4 := AssertExistsAndLoadBean(t, &User{ID: 4}).(*User)
+
+	// Create some reviews
+	expectedReviews := make(map[int64]*PullReviewersWithType)
+	// This one is from fixtures
+	expectedReviews[1] = &PullReviewersWithType{
+		User:              *user1,
+		Type:              ReviewTypeApprove,
+		ReviewUpdatedUnix: 946684810,
+	}
+
+	reviews := []CreateReviewOptions{
```

Comment:
Please don't create reviews since this test should only test the `GetReviwersByPullID` function. Fixtures are the right place for those reviews.